src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -1405,7 +1592,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Cluster Access Control
- *    The Access Control Cluster exposes a data model view of a
+ *
+ * The Access Control Cluster exposes a data model view of a
```

Comment:
Will multiline comments work here? I would have expected the `*` to be continued below if a human was writing these doc-comments.

Reply:
This will work, yes.  I expect that if this file were being run through restyled it would work even better, but right now it's not for various reasons....